### PR TITLE
437 add retention policy to policy page

### LINKED ISSUE #437: Add Retention Policy to policy page
The text of the policy is here (nd only):

https://docs.google.com/document/d/1XO9ioXV8eU-xjOfFriGLJtV5GRqqolNbF2kqTS_YaYI/edit

### PR DESCRIPTION
**Pull request by laritakr**
## Claiming issue 437

@bbe85b725846e9517deb5f24273cf38eff283104

relates to #437

$ ./script/start-issue 437

[skip ci]

## Add Retention and Review Policy page

@d71784e9ddd6bc9a238b0a74f1b59e14e2442381

Left out sentence referring to Revision and Withdrawal Policy, and Content
Format Preservation and Migration Policies since they do not yet exist.

## Closing issue 437

@b1f938f97f4d0963b2f479849868c385415324f4

Closes #437

$ ./script/close-issue 437

### CODE REVIEW

File: app/assets/stylesheets/modules/policies.css.scss.erb [hunk 1]
```diff
@@ -22,7 +22,7 @@
   border: 1px solid $gray-faint;
   box-sizing: border-box;
   margin: 2em auto 0;
-  max-width: 36em;
+  max-width: 38em;
```
Comment (jeremyf) [inline app/assets/stylesheets/modules/policies.css.scss.erb]: What is the reason for this change?

File: app/views/static_pages/policies-retention-review.html.erb [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  <ul>
+    <li>Have past usage rates via CurateND been high or low?</li>
+    <li>Is content likely to be used, or continue to be used in the future (i.e., has the content been superseded by other scholarship)?</li>
+    <li>Is the content deemed especially rare, ephemeral, unique, or significant?</li>
+  </ul>
+
+  <p><strong>Please note: </strong>Any work with metadata marked private (and thus with private content) may be removed if further preservation requirements are not documented or cannot be proven by content owners.
+  <br><strong>Also note: </strong>CurateND will make reasonable efforts to contact content owners based on available information.  If content owners cannot be reached, the Hesburgh Libraries will make the final determination as to whether content should continue to be preserved.</p>
+
+  <h4>Size Threshhold</h4>
```
Comment (jeremyf) [inline app/views/static_pages/policies-retention-review.html.erb]: This should be `<h4>Size Threshold</h4>`

File: app/assets/stylesheets/modules/policies.css.scss.erb (hunk 1 above)
Comment (laritakr) [inline app/assets/stylesheets/modules/policies.css.scss.erb]: Box was widened to prevent the new page title from wrapping.
